iml-agent/src/cli.rs

```diff
@@ -235,6 +235,14 @@ pub enum NtpClientCommand {
     IsConfigured,
 }
 
+#[derive(Debug, StructOpt)]
+pub enum SnapshotCommand {
+    /// Create a snapshot
+    Create(lustre::snapshot::Create),
+    /// Destroy the snapshot
+    Destroy(lustre::snapshot::Destroy),
+}
```

Comment:
This is going to have to move to `wire-types` with a compile flag for `StructOpt` support.